sql/moz-fx-data-shared-prod/telemetry_derived/firefox_use_counters_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Firefox Use Counters
+description: |-
+  Contains use counter rates by day, country, version, and platform; only shows data for those with at least 5000 clients reporting that day.
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: use_counter_analysis
+bigquery:
+  time_partitioning:
+    type: day
+    field: 'received_date'
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields: ['metric']
```

Comment:
I'd suggest you use this style / formatting:
https://github.com/mozilla/bigquery-etl/blob/a500501063ddbe9b4992811422fe2ce34c2e0c12/sql_generators/serp_events/templates/metadata.yaml#L32C1-L35

Reply:
thanks for the example, updated  the metadata yaml file